spec/system/prior_authority/application_lists_spec.rb

```diff
@@ -23,7 +23,7 @@
       expect(page).to have_content 'BBBBB'
     end
 
-    within '#assessed' do
+    within '#reviewed' do
```

Comment:
I had created a more exhaustive test for each of the "reviewed" state just to black box test all in one go

```
#setup
    create(:prior_authority_application, laa_reference: 'LAA-CCCC1', status: 'granted', updated_at: 3.days.ago)
    create(:prior_authority_application, laa_reference: 'LAA-CCCC2', status: 'sent_back', updated_at: 3.days.ago)
    create(:prior_authority_application, laa_reference: 'LAA-CCCC3', status: 'part_grant', updated_at: 3.days.ago)
    create(:prior_authority_application, laa_reference: 'LAA-CCCC4', status: 'rejected', updated_at: 3.days.ago)
    create(:prior_authority_application, laa_reference: 'LAA-CCCC5', status: 'auto_grant', updated_at: 3.days.ago)
```

```
    within '#reviewed' do
      expect(page)
        .to have_content('CCCC1')
        .and have_content('CCCC2')
        .and have_content('CCCC3')
        .and have_content('CCCC4')
        .and have_content('CCCC5')
    end
```